projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -94,18 +94,31 @@ module.exports = async function (...args) {
 		buildSoy();
 	}
 
-	runBabel(
-		BUILD_CONFIG.input,
-		'--out-dir',
-		BUILD_CONFIG.output,
-		'--source-maps'
-	);
+	const disableOldBuild =
+		FEDERATION_CONFIG && FEDERATION_CONFIG.disableOldBuild;
+
+	if (!disableOldBuild) {
```

Comment:
So you said this isn't permanent; what will the final state be?

If it's only temporary this is ok, but straight away, having a boolean that is only ever checked in negated form is a bit of a smell (eg. double negative "not disabled" instead of "enabled").

Reply:
It's negative because the canonical way to build things has always been executing babel and bundler, so disabling it is the exception. Of course we can turn it into enable and default it to true, but it's a bit strange that you have to enable something that has always been there...

Nevertheless, the final state is that all this will be removed, because we won't need to run the bundler or babel alone.

Reply:
> It's negative because the canonical way to build things has always been executing babel and bundler, so disabling it is the exception.

I'm not saying you should _restructure_ the code; I'm saying your variable _name_ is bad, because literally _every time you use it_, you express it in a double-negative form. As a native English speaker, chained negatives sound "bad", and they require more cognitive effort to understand.

ie. if this were to stay in the code, the `if` would remain in exactly the same place, but you'd flip the name and drop the `!`; eg:

```js
const classicBuild = !FEDERATION_CONFIG || !FEDERATION_CONFIG.disableOldBuild;
//                                                            ^-------------^
//                                                                    |   
//                                         ignoring this for now, which is also "bad"    

if (classicBuild) {
  // etc...
```

All moot, of course, because this code will go away, but the point about double-negatives being undesirable is good to bear in mind for future occasions.

Reply:
Ah OK. That makes more sense, yes :+1: . 

Anyway, I'll change it in a future PR. It's not that I don't want to change the code, but every time we make any small change I have to do a full test cycle which includes an `ant all`, so unless we really need to change something before merging the PR I prefer to delay it... :grimacing: 

Also, in this case, we should be removing this soon :pray: . If we need it in the end, we will do a full refactor of the new things so that they stay coherent and clean (this is not the only temporary fix we will be merging, I suspect...